src/_data/sites/zh-hans.yml

```diff
@@ -16,6 +16,10 @@ language:
   title: Simplified Chinese
 locale: zh-hans
 hostname: zh-hans.eslint.org
+locals:
+  docs_latest: zh-hans-docs.netlify.app
```

Comment:
This version shows HEAD in the version selector, while what we want to show is the latest translated version (currently v8.25.0)?

https://deploy-preview-357--zh-hans-eslint.netlify.app/docs/latest/

![image](https://user-images.githubusercontent.com/44349756/197301183-868aa971-7b71-4b51-842a-89f9aae8e58a.png)

Selecting v8.25.0 reloads the same page, and there's again HEAD in the version selector.

Reply:
Oh, now I see https://github.com/eslint/zh-hans.docs.eslint.org/issues/76. For some reason, I haven't been getting notifications for the past few hours.